classes/Carrier.php

```diff
@@ -1244,7 +1247,7 @@ public static function getIdTaxRulesGroupByIdCarrier($id_carrier, Context $conte
     public function setTaxRulesGroup($id_tax_rules_group, $all_shops = false)
     {
         if (!Validate::isUnsignedId($id_tax_rules_group)) {
-            die(Tools::displayError());
+            die(Tools::displayError('Tax rule ID is invalid.'));
```

Comment:
I think using _Parameter "id_tax_rules_group" is invalid_ here is better and keep consistency.